tailnet/proto/tailnet.proto

```diff
@@ -131,18 +130,15 @@ message Netcheck {
 
 	int64 PreferredDERP = 13;  // 0 for unknown
 
-	map<int64, google.protobuf.Duration> RegionLatency = 14;
-	map<int64, google.protobuf.Duration> RegionV4Latency = 15;
-	map<int64, google.protobuf.Duration> RegionV6Latency = 16;
+	map<int64, google.protobuf.Duration> RegionV4Latency = 14;
+	map<int64, google.protobuf.Duration> RegionV6Latency = 15;
 
 	message NetcheckIP {
 		string hash = 1;
 		IPFields fields = 2;
 	}
 	NetcheckIP GlobalV4 = 17;
 	NetcheckIP GlobalV6 = 18;
-
-	google.protobuf.BoolValue CaptivePortal = 19;
```

Comment:
This was removed because captive portal checks are disabled in Coder's tailscale fork anyways